app/src/routes/v1/submission.ts

```diff
@@ -54,7 +54,7 @@ router.get(
 
 /** Creates or updates an intake and set status to Draft */
 router.put(
-  '/draft',
+  '/draft/',
```

Comment:
Was the trailing slash required for something?

Reply:
I just noticed that all of our other endpoints had trailing ones so I added it